app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -36,25 +36,24 @@ keywords:
   - Statement
   - Comparability
 ---
-
-Overseas teachers enrich the lives of students and add value to schools and the wider community. The [Department for Education (DfE)](https://www.gov.uk/government/organisations/department-for-education) wants to help Ukrainian refugees in any way we can, and we support and encourage schools and training providers to recruit Ukrainian teachers and teacher trainees. 
+The [Department for Education (DfE)](https://www.gov.uk/government/organisations/department-for-education) supports and encourages Ukrainian teachers and teacher trainees seeking jobs and training places. 
```

Comment:
@laurahermionetennant92 could we switch this and have Ukrainian teacher trainees and teachers?

Reply:
Yes :)